apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPage.tsx

```diff
@@ -46,29 +50,51 @@ export const EditPage = () => {
         }
     };
 
+    const handleSaveDraft = async () => {
+        await savePageAsDraft(token, Number(pageId))
+            .then((response) => {
+                console.log(response);
+                setAlertMessage('Page successfully saved as Draft');
+                setAlertType('success');
+            })
+            .catch((error) => {
+                setAlertMessage(error.body.message);
+                setAlertType('error');
+            });
+    };
+
     return (
         <PageBuilder page="edit-page">
             {page ? (
                 <div className="edit-page">
                     <PagesBreadcrumb currentPage={page.Name} />
                     <div className="edit-page__header">
-                        <EditPageHeader page={page} />
+                        <EditPageHeader page={page} handleSaveDraft={handleSaveDraft} />
                         <EditPageTabs tabs={tabs} active={active} setActive={setActive} />
                     </div>
                     <div className="edit-page__container">
-                        {page.pageTabs[active] ? (
-                            <EditPageContentComponent content={page.pageTabs[active]} onAddSection={handleAddSection} />
-                        ) : null}
+                        <div className="edit-page__content">
+                            {alertMessage !== '' ? <AlertBanner type={alertType}>{alertMessage}</AlertBanner> : null}
```

Comment:
i dont think you need !== '', as an empty string evaluates to falsy

Reply:
alertMessage default is '', only displaying when there is a message to display